internal/bminventory/inventory_test.go

```diff
@@ -2702,15 +2702,19 @@ var _ = Describe("cluster", func() {
 				err = db.Model(&models.Host{ID: &masterHostId3, ClusterID: &clusterID}).UpdateColumn("free_addresses",
 					makeFreeNetworksAddressesStr(makeFreeAddresses("10.11.0.0/16", "10.11.12.15", "10.11.12.16"))).Error
 				Expect(err).ToNot(HaveOccurred())
-				mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(1)
 			})
 
 			mockSuccess := func(times int) {
 				mockClusterUpdateSuccess(times, 3)
 			}
 
+			mockClusterUpdatability := func(times int) {
+				mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(times * 1)
```

Comment:
why not
```suggestion
				mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(times)
```
?

Reply:
Copied over from:
https://github.com/openshift/assisted-service/blob/2650b97301937a647c2a26e9643faa8499d26b23/internal/bminventory/inventory_test.go#L4832-L4834

Will change it as you proposed 👍

Reply:
Done.